### ISSUE TITLE
Metric pages show almost no info

### ISSUE THREAD

**Issue by wlach** [mozilla/glean-dictionary#66]
*This issue is intended as an onboarding task for potential outreachy applicants. Please do not work on it unless you have completed the initial qualification task and it has been assigned to you.*

Despite having a bunch of metadata in them, we show almost nothing about the metrics on the metric page.

For a start, let's show everything the earlier glean dictionary prototype did:

![image](https://user-images.githubusercontent.com/20569/94936132-d8487000-049b-11eb-8dbc-3d104ffc995e.png)
https://glean-dictionary.netlify.app/?metric=media_state_play

The code that needs to be modified is here:

https://github.com/mozilla/glean-dictionary/blob/main/src/pages/MetricDetail.svelte

All the information we want to display *should* already be extracted. If you have the server running locally, go to e.g.:

http://localhost:5000/data/mach/metrics/mach.system.memory.json

This is the dataset corresponding to:

http://localhost:5000/#!/apps/mach/metrics/mach.system.memory

You can skip the "help" parts of this dialog for now. We'll tackle that in a separate issue. Also, don't worry about styling the component too much-- can just use a table for now (like we do in the table view already: http://localhost:5000/#!/apps/mach/tables/usage)

**Comment by meghajain-1711** [comment]
Hi @wlach So far I have added the following info ont he page through my code:  

mach.system.memory
Amount of system memory.
relevant bugs : 1
send in pings : usage
lifetime application
disabled : false
data reviews 1
version : 0

And the page looks something like this : 
![image](https://user-images.githubusercontent.com/59704142/95591034-6a381600-0a64-11eb-91fc-11930f755ec9.png)

The keys have been taken based on the keys showing in the netlify.app page. 
More info link has been added for lifetime as of now. Data Reviews and Bugs the no links to the hyperlink of bug as mentioned in the json. 

Kindly review and let me know if it looks headed in the right direction. 

Thanks :)
